bentoml/_internal/runner/runner_handle/remote.py

```diff
@@ -41,6 +41,15 @@ def __init__(self, runner: Runner):  # pylint: disable=super-init-not-called
     def _remote_runner_server_map(self) -> dict[str, str]:
         return BentoMLContainer.remote_runner_mapping.get()
 
+    @property
+    def runner_timeout(self) -> int:
+        "return the configured timeout for this runner."
+        runner_cfg = BentoMLContainer.runners_config.get()
+        if self._runner.name in runner_cfg:
+            return runner_cfg[self._runner.name]["timeout"]
+        else:
+            return runner_cfg["timeout"]
+
```

Comment:
@bojiang This is how we get the timeout for each runner client

Reply:
Is there a case that the `runner_name` exists in the config but didn't provide a `timeout` ?

Reply:
nope, the BentoML config will copy over the default config to the `runner_name` so that situation will not happen.